sdk/monitor/azure-monitor-opentelemetry/tests/configuration/test_configure.py

```diff
@@ -199,15 +199,17 @@ def test_setup_resources(self):
         _setup_resources()
         self.assertEqual(
             os.environ["OTEL_EXPERIMENTAL_RESOURCE_DETECTORS"],
-            "azure_app_service,azure_vm"
+            # TODO: Change back to "azure_app_service,azure_vm" after VM Resource Detector fix
```

Comment:
Please link the [issue](https://github.com/Azure/azure-sdk-for-python/issues/33295) in the TODO

Reply:
Done